<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>758</width>
    <height>368</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>家庭光照管理</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>160</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>A-OTF Akashi Std</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>卧室A灯</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>160</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>A-OTF Akashi Std</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>客厅灯</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>240</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>A-OTF Akashi Std</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>卫生间灯</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>240</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>A-OTF Akashi Std</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>厨房灯</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>240</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>A-OTF Akashi Std</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>书房灯</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>160</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>A-OTF Akashi Std</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>卧室B灯</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>240</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>A-OTF Akashi Std</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>阳台灯</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>160</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>A-OTF Akashi Std</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>餐厅灯</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>571</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>A-OTF Akashi Std</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>当前室外光照：330LX</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>A-OTF Akashi Std</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>当前室内光照：278LX</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>758</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
